app/services/foreman_ansible/imprt_playbooks.rb

```diff
@@ -0,0 +1,14 @@
+
+module ForemanAnsible
+  class PlayBookImporter
+
+    def import_playbook
+      # Read a playbook example from github, needs to be fixed to read playbooks from certain directory
+      uri = URI("https://raw.githubusercontent.com/theforeman/forklift/master/playbooks/luna_demo_environment.yml")
+      file = Net::HTTP.get(uri)
+      template = JobTemplate.import_raw(File.read(file))
```

Comment:
`import_raw` can certainly create a job template from a file, it is primarily intended for [text representation of a job template](https://github.com/theforeman/foreman_ansible/blob/master/app/views/foreman_ansible/job_templates/capsule_upgrade_-_ansible_default.erb), which also includes metadata. 

Since playbooks do not have metadata, you will need to create it. Metadata is just yaml enclosed in `<#%` and `%>` tags, so [YAML](https://ruby-doc.org/stdlib-2.5.1/libdoc/yaml/rdoc/YAML.html) module might be useful.
Adding name, snippet, model, job_category, provider_type and kind keys would be plenty for now. 
Once you have metadata, it will be a part of  job template content together with playbook file.